gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -729,29 +817,56 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
     case PROP_MODEL:
     {
       const gchar *model_files = g_value_get_string (value);
+      GstTensorFilterProperties _prop;
 
       g_assert (model_files);
+      _prop.model_files = NULL;
+
+      if (prop->fw_opened) {
+        /** Store a copy of the original prop in case the reload fails */
+        memcpy (&_prop, prop, sizeof (GstTensorFilterProperties));
+        _prop.model_files =
+            (const gchar **) g_strdupv ((gchar **) prop->model_files);
```

Comment:
if you allocate new memory for ```_prop.model_files```, you should free it somewhere.
You've simply done ```_prop.model_files = NULL;``` at LINE 823 instead.

Reply:
Memory for `_prop.model_files` is freed at LINE 857 (when reload is successful).
If reload is failed, then `prop.model_files` is freed and `_prop.model_files` is used.